include/nbl/builtin/hlsl/device_capabilities_traits.hlsl

```diff
@@ -5,14 +5,12 @@
 #define _NBL_BUILTIN_HLSL_DEVICE_CAPABILITIES_TRAITS_INCLUDED_
 
 #include <nbl/builtin/hlsl/member_test_macros.hlsl>
+#include "nbl/video/SPhysicalDeviceLimits.h"
+#include "nbl/video/SPhysicalDeviceFeatures.h"
```

Comment:
IMO HLSL headers don't need to know about this

Reply:
Done